internal/client.go

```diff
@@ -381,6 +381,8 @@ func (c *rmqClient) Start() {
 			}
 		})
 	})
+	//set c.close to false, or it could send only one message
+	c.close = false
```

Comment:
I'm not sure if it result in other problems when re-start, do you have taken a full testing?